examples/post-process/bloom.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Unreal Bloom Effect
+ * Implementation for A-Frame
+ * Code modified from Akbartus's UnrealBloomPass.js
+ * https://github.com/akbartus/A-Frame-Component-Postprocessing
+ */
+
+import AFRAME from "aframe";
+import { EffectComposer } from "three/addons/postprocessing/EffectComposer.js"; // This uses the same three instance.
+import { RenderPass } from "three/addons/postprocessing/RenderPass.js"; // This uses the same three instance.
+import { UnrealBloomPass } from "three/addons/postprocessing/UnrealBloomPass.js"; // This uses the same three instance.
+
+/////////////////////////////////////
+// A-FRAME COMPONENT: POST-PROCESSING //
+/////////////////////////////////////
+AFRAME.registerComponent("bloom", {
+  schema: {
+    threshold: {
+      type: "number",
+      default: 0,
+    },
+    strength: {
+      type: "number",
+      default: 0.4,
+    },
+    radius: {
+      type: "number",
+      default: 0,
+    },
+    exposure: {
+      type: "number",
+      default: 1,
+    },
+  },
+  init: function () {
+
+    this.scene = this.el.object3D;
+    this.renderer = this.el.renderer;
+    this.camera = this.el.camera;
+    this.bound = false;
+
+    var that = this;
+
+    that.evaluateEffect();
+    that.bind();
```

Comment:
This can all be simplified to only calling `this.bind()`. The assignments also happen in `evaluateEffect` and `update` is always called right after `init`, so no need to call it from `init`.